CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# How to contribute
+
+The NEST simulator is a scientific tool and as such it is never ready and constantly changing to meet the needs of novel neuroscientific endeavors. Here you find the the most important information on how you can contribute to NEST. This document is an excerpt from our [developer space](https://nest.github.io/nest-simulator/), which provides more detailed information.
+
+## Getting started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* The development workflow is based purely on pull requests. This [article](https://nest.github.io/nest-simulator/development_workflow) gives information on how to work with git and GitHub, if you are new to it.
+
+## Making Changes
+
+* Create a topic branch from NEST master in your fork. Please avoid working directly on the `master` branch.
```

Comment:
Add a reference to https://github.com/nest/nest-simulator/issues/31 to explain why. Or even better integrate that information into the developer space!